common/Inc/stm32xx_hal_i2c.h

```diff
@@ -0,0 +1,15 @@
+#if defined(STM32F4xx)
+    #include "stm32f4xx_hal_i2c.h"
+    #include "stm32f4xx_hal_fmpi2c.h"
+    #include "stm32f4xx_hal_rcc.h"
+    #include "stm32f4xx_hal_rcc_ex.h"
+    #include "stm32f4xx_hal_gpio.h"
+
+#elif defined(STM32L4xx)
+    #include "stm32l4xx_hal_i2c.h"
+    #include "stm32l4xx_hal_fmpi2c.h"
+    #include "stm32l4xx_hal_rcc.h"
+    #include "stm32l4xx_hal_rcc_ex.h"
+    #include "stm32l4xx_hal_gpio.h"
+    
+#endif
```

Comment:
What was the reason for having this? Won't these be compiled in based on `L4` and `F4` already?

Reply:
Chat is this real.

Reply:
Turns out I never used this header so it has been removed.